Sources/IBMCloudAppID/Token.swift

```diff
@@ -36,8 +36,8 @@ public class Token {
         return payload["exp"].double
     }
 
-    public var aud: String? {
-        return payload["aud"].string
+    public var aud: Array<String>? {
+        return payload["aud"].object as? [String]
```

Comment:
We should check if this is an `string` in which case convert it to an array (V3 tokens)